environment.yml

```diff
@@ -6,6 +6,7 @@ dependencies:
   - pandas=2.2.3
   - scikit-learn=1.5.2
   - ucimlrepo=0.0.7
-  - ipykernel=6.29.5
-  - numpy=2.1.3
-  - scipy=1.14.1
\ No newline at end of file
+  - scipy=1.14.1
+  - python=3.11
+  - pandera=0.21.0
+  - deepchecks=0.3.2
```

Comment:
Just double-checking for the pinned version of `deepchecks`, any particular reason of using `0.3.2`? `0.18.1` should be the latest release, if there are any compatibility issue with this version, please disregard this comment.

Reply:
`0.3.2` was the version that conda automatically installed into our environment with the other packages, but I can try it with the newer version.